.travis.yml

```diff
@@ -40,6 +45,7 @@ deploy:
       repo: rails-girls-summer-of-code/rgsoc-teams
 addons:
   postgresql: '9.5'
+  chrome: stable
```

Comment:
Would that mean that we don't need to install it manually in `before_install`?

Reply:
As I said, I threw this together pretty quickly just to show how it could eventually look like 🙈 Sorry for the messy state this is in.
For the installation, I followed the [Travis Docs](https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-the-Chrome-addon-in-the-headless-mode) (basically coming from [this Thoughtbot blogpost](https://robots.thoughtbot.com/headless-feature-specs-with-chrome) and the setups I use at work).

As of the docs, both would be necessary, but I think one should definitely do some more experiments there, since it looks suspiciously complex 👀